### PR TITLE
create faq section

### PR DESCRIPTION
**Pull request by IvanSidorenkov**

### CODE REVIEW

File: src/components/faqElement/FaqElement.module.scss [hunk 1]
```diff
@@ -1,39 +1,99 @@
...
+    height: 100%;
+    margin: 30px 60px 40px 30px;
+    font-size: 18px;
+    line-height: 28px;
+  }
+
+  30% {
+    opacity: 0;
+    height: 100%;
+    margin: 30px 60px 40px 30px;
```
Comment (AndreiLavrov) [inline src/components/faqElement/FaqElement.module.scss:59]: why did you duplicate this and another code?)

File: src/components/faqElement/FaqElement.tsx [hunk 2]
```diff
@@ -1,25 +1,28 @@
 import React, { useState } from 'react';
+import Image from 'next/image';
 
 import styles from './FaqElement.module.scss';
```
Comment (AndreiLavrov) [inline src/components/faqElement/FaqElement.tsx:4]: Destructuring

File: src/sections/faq/Faq.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+import { FAQ } from '@constants/faqData';
+import styles from './Faq.module.scss';
```
Comment (AndreiLavrov) [inline src/sections/faq/Faq.tsx:5]: Destructuring

File: src/sections/faq/Faq.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          <p className={styles.description}>
+            Напиши нам и ваш личный менеджер ответит в любое разумное время.
+          </p>
+        </div>
+        <div className={styles.managerContacts}>
+          <div className={styles.managerWrapper}>
+            <Image className={styles.managerImage} src={managerImage} alt="manager" />
+            <p className={styles.managerName}>Менеджер Вика</p>
+          </div>
+          <Link href={''} className={styles.telegramWrapper}>
```
Comment (ssavitski) [inline src/sections/faq/Faq.tsx:35]: No need in curly braces

**Review by ssavitski — APPROVED** [review]

**Review by AndreiLavrov — APPROVED** [review]

**Pull request merged**